src/flavors/kittitas-vsp/config.json

```diff
@@ -1194,7 +1194,7 @@
             "//": "=============== IRRIGATION SCHEDULING ========================",
             "name": "irrigation_scheduling",
             "type": "big_radio",
-            "prompt": "Irrigation scheduling – monitoring soil moisture and adjusting water applications accordingly",
+            "prompt": "Irrigation scheduling - monitoring soil moisture and adjusting water applications accordingly",
```

Comment:
These diffs are replacing the unicode characters with their ascii equivalents. Not a big deal.